client/src/components/maps/MyMap.js

```diff
@@ -5,13 +5,11 @@ import { Hikes } from '../Hikes.js'
 import { Form } from '../Form.js'
 import { Button } from '../Button.js'
 import { GoogleMap, useLoadScript, Marker, InfoWindow } from "@react-google-maps/api"
-import * as hikeData from '../../dummyHikes.json'
+//import * as hikeData from '../../dummyHikes.json'
```

Comment:
Are comments needed on an import that isn't being used?

Reply:
Keeping temporarily, just until we know it's not needed and changes all working once PR accepted